### PR TITLE
Add Profile Color.

### PR DESCRIPTION
**Pull request by AllianceSky-PS**

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
Before I review is this tested, @AllianceSky-PS

**Comment by AllianceSky-PS** [comment]
@DeathlyPlays not tested yet. ( I'll test tomorrow, gotta sleep now )

**Comment by DeathlyPlays** [comment]
Oh ok

P.S. You should join the Server Hangout Discord so I can give you the Developer role.

Link: https://discord.gg/hsAqGBT

**Comment by AllianceSky-PS** [comment]
@DeathlyPlays I've already joined ig :3.

**Comment by DeathlyPlays** [comment]
What's your Discord name

**Comment by DeathlyPlays** [comment]
Ignore that TypeScript fucked up the server's eslint

**Comment by AllianceSky-PS** [comment]
Alliance Sky ( lost the old acnt ).
@DeathlyPlays Why don't you just disable typescript for now?

**Comment by DeathlyPlays** [comment]
Im tempted to

File: server-plugins/profile.js [hunk 1]
```diff
@@ -634,28 +703,28 @@ exports.commands = {
 				profile += `${background(toId(username))} ${showBadges(toId(username))}`;
 				profile += `<img src="${avatar}" height="80" width="80" align="left">`;
 				profile += `&nbsp;<font color="#24678d"><strong>Name:</strong></font> ${Server.nameColor(username, true)}&nbsp; ${getFlag(toId(username))} ${showTitle(username)}<br />`;
```
Comment (DeathlyPlays) [inline server-plugins/profile.js]: We could also set the color where it says:
`<font color="24678d">`
But I'll probably do that so don't worry I guess LOL

Comment (AllianceSky-PS) [reply server-plugins/profile.js]: @DeathlyPlays I didn't intended to change the color of Name:, Group:, Bucks: etc. I just wanted to change text color of it's content like amount of bucks etc. anyway i changed it and tested.

Comment (DeathlyPlays) [reply server-plugins/profile.js]: No worries. It was just an idea I might add on later

**Comment by DeathlyPlays** [comment]
@AllianceSky-PS so its tested now?

**Comment by AllianceSky-PS** [comment]
@DeathlyPlays Yeah it is. But i recommend you to test it yourself once before merge.

**Comment by DeathlyPlays** [comment]
Ah ok because I can't at the moment

**Pull request merged**